tests/playwright/src/model/pages/containers-page.ts

```diff
@@ -175,4 +175,13 @@ export class ContainersPage extends MainPage {
       return this;
     });
   }
+
+  async getContainerImage(name: string): Promise<string> {
+    const container = await this.getContainerRowByName(name);
+    const image = await container?.getByRole('cell').nth(5).textContent();
```

Comment:
don't really like the magic number here. is there no ARIA for this? might be worth committing that as a PR to PD

Reply:
Good path!